modules/@apostrophecms/job/index.js

```diff
@@ -240,7 +241,7 @@ module.exports = {
               setResults (_results) {
                 results = _results;
               }
-            });
+            }, notif);
```

Comment:
probably the same thing. Pass as part of an object. it could be read as `info` or something like that on the receiving side.

Reply:
Passing the info object here!